lib/model/binding.dart

```diff
@@ -66,6 +68,34 @@ abstract class ZulipBinding {
     _instance = this;
   }
 
+  /// Provides device and operating system information,
+  /// via package:device_info_plus.
+  ///
+  /// The returned Future resolves to null if an error is
+  /// encountered while fetching the data.
+  Future<BaseDeviceInfo?> get deviceInfo;
```

Comment:
I think the "This wraps" line in the existing dartdoc is helpful — it helps tie this member to a specific part of the plugin's API.